app/controllers/api/v0/api_controller.rb

```diff
@@ -29,6 +29,49 @@ def auth_results
     render json: { status: "ok" }
   end
 
+  def user_qualification_data
```

Comment:
Shouldn't this be moved to a `users_controller`?

I would expect that routes for `persons` live in a `persons_controllers` and routes for `users` to live in a `users_controller`

Reply:
An early commit had it in a `results_controller`, we moved it here for reasons I can't quite recall. I'll check with Gregor on Friday.